crates/task-impls/src/network.rs

```diff
@@ -278,10 +355,10 @@ impl<TYPES: NodeType, COMMCHANNEL: CommunicationChannel<TYPES>>
             }
             HotShotEvent::Shutdown => {
                 error!("Networking task shutting down");
-                return Some(HotShotTaskCompleted::ShutDown);
+                return Some(HotShotTaskCompleted);
             }
-            event => {
-                error!("Receieved unexpected message in network task {:?}", event);
+            _event => {
+                // error!("Receieved unexpected message in network task {:?}", event);
```

Comment:
I think this error message may be helpful to ensure the event filter isn't missing any new events. "unexpected message" should be "unexpected event" though.

Reply:
We are not logging in other tasks though, so I'm not sure if we want to do this here. Maybe all places or none.

Reply:
I think you're right that we should log here, I think I turned it off because I was debugging and issue and turned off filtering see if that was the cause, which meant this line was hit repeatedly.  It should be fixed now, so I'll uncomment this line.